Makefile

```diff
@@ -0,0 +1,34 @@
+clean:
+	./hack/virt-delete-sno.sh || true
+	rm -rf mydir
+
+generate:
+	mkdir -p mydir
+	cp ./install-config.yaml mydir/
+	OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="quay.io/openshift-release-dev/ocp-release:4.6.5-x86_64" ./bin/openshift-install create manifests --dir=mydir
+	cp ./sno_manifest.yaml mydir/openshift/
```

Comment:
There have been previous proposals to give the installer a `Makefile`, but they were rejected as "not all that useful, vs. calling `hack/...` scripts directly".  See discussion in #1317 and links from there to even earlier discussion.

Narrowly for this PR, if there was an installer `Makefile` and it had single-node-specific recipes like this `sno_manifest.yaml` copy, I'd expect it to be using a single-node-specific rule name (e.g. `generate-single-node-ignition-configs` or some such).

Reply:
Removing the Makefile as it was just used for testing